doc/entropy-seems-wrong.md

```diff
@@ -0,0 +1,107 @@
+
+
+Hacking Techniques
+------------------
+
+Really, the password strength meter does not measure the technique used to generate a password.  It only measures the password and what it would take for the password to be compromised.  Explained another way: your password generator may have a truly random technique of making your password but that doesn't help if it generates "abcd1234".
+
+
+### Common Passwords
+
+People collect lists of passwords that have been successful on other sites.  This will be the first attack that will be performed whenever someone is trying to hack or crack passwords.
+
+Do not reuse passwords.  Use a password manager that was reviewed for security holes and generate unique, long, random passwords everywhere.
+
+
+### Wordlists
+
+After the handful of seconds it takes to try a million common passwords, the very next attack is typically against wordlists.  People collect words, slang and misspellings from all of the languages, then they feed these wordlists into their password crackers.
+
+Do not use a single word from a wordlist.  Use a password manager that was reviewed for security holes and generate unique, long, random passwords everywhere.
+
+
+### Mutation Engines
+
+Processing wordlists takes some time, but not a very long amount of time.  Hackers will still be quite interested in cracking passwords even after they run out of words.  One of the techniques that may be employed is to use a mutation engine on the wordlist in order to generate more passwords to check.  The mutation engine would change capitalization, add numbers to the beginning or the end, add symbols and possibly even concatenate words.
+
+This increases the invested time dramatically, but also yields good results.  How many times have you thought about using a password with a year or other number after it?
+
+It is also an option to pass entries through filters that apply other alterations.  You can pluralize words or conjugate verbs differently.  It can be translated ROT13, pig latin or leetspeak.
+
+
+### Algorithm-specific Attacks
+
+A bit of knowledge goes a long way.  Once I worked for a company that required passwords that were exactly 8 characters.  You must have at least one capital letter, one lowercase letter, one number and one symbol.  At first blush you may think that you have a possible 95 characters for each position and we need a total of 8 characters, so 95^8 possibilities (6.6E15).
+
+Remember that "a bit of knowledge goes a long way" from earlier?  Some more information you've obtained is that the list of symbols that are allowed is either `.` or `$`.  Only those two are permitted.  Not even spaces are allowed.  So your character set size drops to 64, and the total is 64^8 (2.8E14).  We've reduced the total number of passwords down to about a 1/23 of what it was previously.
```

Comment:
Where the quote in the first sentence may need ... before and after... in the quotes.